toolchains/docker/readme.md

```diff
@@ -28,6 +28,10 @@ register_toolchains(
 )
 ```
 
+The docker toolchain automatically configures itself to use the docker in your
```

Comment:
These rules by default use the docker in your system path. To override...

Reply:
Done